include/haproxy/global-t.h

```diff
@@ -26,6 +26,12 @@
 #include <haproxy/buf-t.h>
 #include <haproxy/freq_ctr-t.h>
 
+#ifdef __linux__
+#ifndef IPPROTO_MPTCP
```

Comment:
if you only use it in one .c file, maybe best to declare it there (`src/cfgparse.c`), just before using it.